services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -127,9 +128,15 @@ object WorkflowQueryKey {
       val nels: List[ErrorOr[String]] = values map { v => {
         allowedValues.contains(v).fold(v.validNel[String], v.invalidNel[String])
       }}
-      sequenceListOfValidatedNels("Unrecognized values", nels)
+      sequenceListOfValidatedNels(s"Keys should be from ${allowedValues.toString}. Unrecognized values", nels)
```

Comment:
think you can just do `$allowedValues` without braces and `toString`.

Reply:
💯 correct! Removed them. Thanks! :)